.travis.yml

```diff
@@ -14,6 +14,8 @@ env:
     - NPROC_MAX=8 BUILDTEST_MCU_GROUP=cortex_m4
 
 before_install:
+    - source ./dist/tools/pr_check/check_labels.sh
+    - test "$BUILDTEST_MCU_GROUP" == "static-tests" || check_gh_label "Ready for CI build" || exit 1
```

Comment:
`==` should be `=`, see `man test`. (My fault, I made the mistake in my previous comment)